frontend/projects/valtimo/layout/src/lib/components/widget-action-button/widget-action-button.component.html

```
@@ -13,8 +13,12 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-@if (!!widgetConfiguration.actions[0] && !!widgetConfiguration.actions[0].navigateTo) {
-  <button class="action-button" cdsButton="ghost" (click)="onNavigateButtonClick(widgetConfiguration.actions[0])">
+@if (getNavigateToUrl(widgetConfiguration.actions[0])) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)